### PR TITLE
more Holy paladin talent fixes

### PR DESCRIPTION
**Pull request by camclark**
More paladin changes to assist build.

Extra shaman issue included.
Looks like some warlock changes snuck in too from lint. (Can remove if preferred) 

Paladin still a WIP but proud to see the hpal report page for the first time.
(Thanks @ToppleTheNun )

image

### CODE REVIEW

File: src/analysis/retail/paladin/holy/modules/dragonflight/tier/Tier29FourSet.tsx [hunk 1]
```diff
@@ -14,15 +14,24 @@ import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
...
+//https://www.wowhead.com/beta/spell=393672/paladin-holy-class-set-4pc
+// Paladin Holy Class Set 4pc
+// Requires Paladin
+// Holy Light, Flash of Light, Light of Dawn, Word of Glory, and Bestow Faith healing is increased by 6% and their critical effects
+// increase the damage or healing of your next Holy Shock by 20%.
+
+// TODO: convert this from Tier28FourSet Tier29FourSet
+// File has just currently been renamed
+
+class Tier29FourSet extends Analyzer {
```
Comment (emallson) [inline src/analysis/retail/paladin/holy/modules/dragonflight/tier/Tier29FourSet.tsx]: you are better off deleting the old tier analyzer and writing a new one from scratch than trying to repurpose it.

File: src/analysis/retail/paladin/holy/CombatLogParser.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ import MightOfTheMountain from './modules/racials/MightOfTheMountain';
 import UntemperedDedication from './modules/talents/UntemperedDedication';
 import MaraadsCastRatio from './modules/talents/MaraadsCastRatio';
 import MaraadsOverheal from './modules/talents/MaraadsOverheal';
-import Tier28FourSet from './modules/shadowlands/tier/Tier28FourSet';
+import Tier29FourSet from './modules/dragonflight/tier/Tier29FourSet';
```
Comment (emallson) [inline src/analysis/retail/paladin/holy/CombatLogParser.ts]: the old tier set should just be deleted, not renamed

File: src/parser/shared/modules/DeathRecapTracker.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { Trans } from '@lingui/macro';
-import { TALENTS_PALADIN } from 'common/TALENTS';
+// import SPELLS from 'common/SPELLS';
```
Comment (emallson) [inline src/parser/shared/modules/DeathRecapTracker.js]: no need to leave this comment here
```suggestion
```

**Review by emallson — CHANGES_REQUESTED** [review]
nice catch with the Hpal tier set being imported in enhancement shaman, but you should delete the old tier analyzer rather than try to repurpose it for an entirely different tier set (unless the tier is very similar)

File: src/analysis/retail/paladin/holy/modules/dragonflight/tier/Tier29FourSet.tsx (hunk 1 above)
Comment (camclark) [reply src/analysis/retail/paladin/holy/modules/dragonflight/tier/Tier29FourSet.tsx]: noted!

**Review by emallson — APPROVED** [review]

**Pull request merged**